engine/launcher/src/main/scala/org/enso/launcher/cli/LauncherApplication.scala

```diff
@@ -99,12 +101,23 @@ object LauncherApplication {
       "Advanced option, use carefully.",
       showInUsage = false
     )
-  private def versionOverride =
+  private def versionOverride = {
     Opts.optionalParameter[SemVer](
       "use-enso-version",
       "VERSION",
       "Override the Enso version that would normally be used."
     )
+  }
+  private def engineLogLevel = {
+    Opts
+      .optionalParameter[LogLevel](
+        "log-level",
+        "LOG-LEVEL",
+        "Sets logging verbosity for the engine. Possible levels are: OFF, " +
+        "ERROR, WARNING, INFO, DEBUG and TRACE."
```

Comment:
If I change this to `(error | warning | info | debug | trace)` the comment can just say `Sets logging verbosity for the engine. Defaults to info.`.

Finally it will look like this in the help text:
```
    [--launcher-log-level (error | warning | info | debug | trace)]
                              Sets logging verbosity for the launcher. If not
                              provided, defaults towarning.
```

Reply:
Looks a bit strange with the long line, but it works!